metadata.json

```diff
@@ -20,7 +20,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 3.4.0"
+      "version_requirement": ">= 4.5.0"
```

Comment:
I already bumped the version to `0.2.0-dev` in a previous PR but never released a 0.2.0.